src/app/screens/LNURLPay/index.tsx

```diff
@@ -517,7 +520,7 @@ function LNURLPay() {
                         )}
                       </div>
                     )}
-                    <div className="mt-2 dark:border-white/10">
+                    <div className="absolute bottom-0 w-full dark:border-white/10">
```

Comment:
Please don't use absolute positioning, as that will lead to other problems. (see screenshot) 

Have a look at other screens where the buttons are already aligned at the bottom (e.g. in some prompt screens)

![image](https://github.com/getAlby/lightning-browser-extension/assets/100827540/db869118-84ab-471d-bde8-b9e79f9c8df9)

Reply:
got it, will make the changes for it!

Reply:
@reneaaron Made the changes can u review it now.
Thanks!